models/public/background-matting-mobilenetv2/README.md

```diff
@@ -0,0 +1,136 @@
+# background-matting-mobilenetv2
+
+## Use Case and High-Level Description
+
+The `background-matting-mobilenetv2` model is a high-resolution background replacement technique based on
+background matting (with MobileNetV2 backbone), where an additional frame of the background is
+captured and used in recovering the alpha matte and the foreground layer. This model is
+pre-trained in PyTorch\* framework and converted to ONNX\* format. More details provided in
+the [paper](https://arxiv.org/abs/2012.07810).
+For details see the [repository](https://github.com/DmitriySidnev/BackgroundMattingV2).
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Background_matting                        |
+| GFlops                          | 6.7419                                    |
+| MParams                         | 5.052                                     |
+| Source framework                | PyTorch\*                                 |
+
+## Accuracy
+
+Accuracy measured on a dataset composed with foregrounds from the HumanMatting dataset and backgrounds from the OpenImagesV5 one with input resolution 1280x720.
+
+| Metric         | Original model | Converted model |
+| -------------- | -------------- | --------------- |
+| Alpha MAD      | 4.32           | 4.35            |
+| Alpha MSE      | 1.0            | 1.0             |
+| Alpha GRAD     | 2.48           | 2.49            |
+| Foreground MSE | 2.7            | 2.69            |
+
+* Alpha MAD - mean of absolute difference for alpha.
+* Alpha MSE - mean squared error for alpha.
+* Alpha GRAD - spatial-gradient metric for alpha.
+* Foreground MSE - mean squared error for foreground.
+
+> **Note**: metrics were measured using the [script](https://github.com/DmitriySidnev/RobustVideoMatting/blob/master/evaluation/evaluate_hr.py).
+> Following the original paper the test dataset obtained from the original one by compositing
+> samples onto 5 random background images. The result dataset can be found [here](https://drive.google.com/file/d/1fjVXmP_gsWZiCsY6BdSxIPIv1tBZWIyK/view?usp=sharing).
```

Comment:
not sure that is good way to redistribute data... Do you have approval for it?

Reply:
Another dataset is being used. This notice removed.